package.json

```diff
@@ -5,19 +5,18 @@
   "browser": "src/browser.js",
   "main": "src/node.js",
   "dependencies": {
+    "browser-level": "^1.0.1",
     "charwise": "^3.0.1",
-    "encoding-down": "^7.1.0",
-    "level-js": "^6.1.0",
-    "leveldown": "^6.1.0",
-    "levelup": "^5.1.1",
+    "classic-level": "^1.2.0",
+    "level-read-stream": "^1.1.0",
+    "memory-level": "^1.0.0",
     "traverse": "^0.6.6"
   },
   "devDependencies": {
     "assert": "^2.0.0",
     "buffer": "^6.0.3",
     "diacritic": "^0.0.2",
-    "level-out": "^1.0.1",
```

Comment:
I never saw this used in the code so I just removed it

Reply:
Yeah- I personally like to use `level-out`as a dev tool to get a quick look at the index, but you are right- it shouldn't be an actual dev dependency